multisig/simulations/sepolia/vip-035/addendum.ts

```diff
@@ -0,0 +1,136 @@
+        parseUnits("3992", 18),
+        parseUnits("1", 18),
+      );
+    });
+  });
+
+  describe("Post-VIP behavior", async () => {
+    before(async () => {
+      await pretendExecutingVip(vip035());
+    });
+
+    it("check owner", async () => {
+      expect(await sfrxETHOracle.owner()).to.equal(sepolia.GUARDIAN);
+    });
+
+    it("check price", async () => {
+      expect(await resilientOracle.getPrice(sfrxETH)).to.be.closeTo(parseUnits("3992", 18), parseUnits("1", 18));
+      expect(await resilientOracle.getUnderlyingPrice(vsfrxETH)).to.be.closeTo(
+        parseUnits("3992", 18),
+        parseUnits("1", 18),
+      );
+    });
+
+    it("should have 6 markets in LST pool", async () => {
+      const poolVTokens = await comptroller.getAllMarkets();
+      expect(poolVTokens).to.have.lengthOf(6);
+    });
+
+    it("should add vsfrxETH to the pool", async () => {
+      const registeredVToken = await poolRegistry.getVTokenForAsset(comptroller.address, sfrxETH);
+      expect(registeredVToken).to.equal(vsfrxETH);
+    });
+
+    it("check ownership", async () => {
+      expect(await vsfrxETHContract.owner()).to.equal(sepolia.GUARDIAN);
+    });
+
+    it("check supply", async () => {
+      const expectedSupply = parseUnits("1.2", 8);
+      expect(await vsfrxETHContract.balanceOf(sepolia.VTREASURY)).to.equal(expectedSupply);
+    });
+
+    it("should set collateral factor to 90% and Liquidation threshold to 93%", async () => {
+      const market = await comptroller.markets(vsfrxETH);
+      expect(market.collateralFactorMantissa).to.equal(parseUnits("0.9", 18));
+      expect(market.liquidationThresholdMantissa).to.equal(parseUnits("0.93", 18));
+    });
+
+    it("check protocol seize share", async () => {
+      expect(await vsfrxETHContract.protocolSeizeShareMantissa()).equals(parseUnits("0.01", 18));
+    });
+
+    it("check vToken", async () => {
+      await checkVToken(vsfrxETH, {
+        name: "Venus sfrxETH (Liquid Staked ETH)",
+        symbol: "vsfrxETH_LiquidStakedETH",
+        decimals: 8,
+        underlying: sfrxETH,
+        exchangeRate: parseUnits("1", 28),
+        comptroller: LST_COMPTROLLER,
+      });
+    });
+    it("check reserve factor", async () => {
+      expect(await vsfrxETHContract.reserveFactorMantissa()).equals(parseUnits("0.2", 18));
+    });
+    it("check IR", async () => {
+      const IR = await vsfrxETHContract.interestRateModel();
+      checkInterestRate(
+        IR,
+        "vsfrxETH_LiquidStakedETH",
+        { base: "0", multiplier: "0.09", jump: "3", kink: "0.4" },
+        BigNumber.from(2628000),
+      );
+    });
+    it("check Pool", async () => {
+      await sfrxETHContract.faucet(parseUnits("10000", 18));
+      await checkIsolatedPoolsComptrollers({
+        [LST_COMPTROLLER]: sepolia.NORMAL_TIMELOCK,
+      });
+    });
+    it("check balance", async () => {
+      expect(await xvsContract.balanceOf(REWARDS_DISTRIBUTOR_XVS)).to.gte(XVS_REWARD_TRANSFER);
+    });
```

Comment:
we are not sending any XVS tokens in this vip. So, we should remove this simulation too.